.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         node:
           - '18.12'
           - '20'
-          - '21'
```

Comment:
We don't want to test against v21?

Reply:
v21 is not an lts version. When 22 is out, v21 is basically deprecated.

Reply:
as you can see, we don't test v19 as well.